Add migration docs (#9)
exercism/v3-launch: migration/add-key-features.md

# Add key features

> In Exercism v3, each track can specify (at most) six key features. Students will use these features to determine whether or not they want to join a track, so the features should highlight the language's best and/or most interesting features.

Reviewer: ```suggestion
In Exercism v3, each track must specify exactly six "key features". Exercism uses these features to highlight the most interesting, unique or "best" features of a language to a student.

```
